@pAnd0rASBG Could you check and fix the javascript codestyle error reported by Drone here https://ci.joomla.org/joomla/joomla-cms/80258/1/20 ?

```
npm run lint:js

> joomla@5.2.2 lint:js
> eslint --config build/.eslintrc --ignore-pattern '/media/' --ext .es6.js,.es6,.vue .

+ npm run lint:testjs

> joomla@5.2.2 lint:testjs
> eslint --config build/.eslintrc --ext .js tests/System


/drone/src/tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js
  22:17  error  Unexpected string concatenation  prefer-template

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
  ```

--- a/tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js
+++ b/tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js
@@ -19,7 +19,7 @@ describe('Test in backend that the action logs', () => {
     cy.doAdministratorLogout();
     cy.doAdministratorLogin();
     cy.visit('/administrator/index.php?option=com_actionlogs&view=actionlogs');
-    cy.contains('User ${Cypress.env('username')} logged in to admin');
+    cy.contains(`User ${Cypress.env('username')} logged in to admin`);
     cy.task('queryDB', 'TRUNCATE #__action_logs');
   });
 

--- a/tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js
+++ b/tests/System/integration/administrator/components/com_actionlogs/Actionlogs.cy.js
@@ -19,7 +19,7 @@ describe('Test in backend that the action logs', () => {
     cy.doAdministratorLogout();
     cy.doAdministratorLogin();
     cy.visit('/administrator/index.php?option=com_actionlogs&view=actionlogs');
-    cy.contains('User ' + Cypress.env('username') + ' logged in to admin');
+    cy.contains('User ${Cypress.env('username')} logged in to admin');
     cy.task('queryDB', 'TRUNCATE #__action_logs');
   });
 

